src/main/java/com/sap/sgs/phosphor/fosstars/data/github/GitHubDataFetcher.java

```diff
@@ -384,19 +367,15 @@ protected Date expiration() {
    * @return The info about local repositories.
    * @throws IOException If something went wrong.
    */
-  protected Map<URL, LocalRepositoryInfo> loadLocalRepositoriesInfo() throws IOException {
-    synchronized (LOCAL_REPOSITORIES_INFO_LOCK) {
```

Comment:
Well, strictly speaking, this method should be synchronized as well. However, since it's only called once during initialization of the class, there should not be any problem. However, if the method may be called multiple times for some reason in the future, there may be problems. If I were you, I would keep synchronization for completeness even if it's not strictly necessary.